mne/filter.py

```diff
@@ -626,6 +626,10 @@ def band_pass_filter(x, Fs, Fp1, Fp2, filter_length='10s',
     xf : array
         x filtered.
 
+    See Also
+    --------
+    low_pass_filter, high_pass_filter
```

Comment:
In signal processing terms resampling can be thought of as low-pass filtering followed by a decimation step (or a zero-insertion step followed by a low-pass)